actionview/lib/action_view/helpers/capture_helper.rb

```diff
@@ -169,14 +169,20 @@ def capture(*, **, &block)
       #   <% content_for :script, javascript_include_tag(:defaults) %>
       #
       # WARNING: <tt>content_for</tt> is ignored in caches. So you shouldn't use it for elements that will be fragment cached.
-      def content_for(name, content = nil, options = {}, &block)
+      def content_for(name, content = nil, flush: false, **options, &block)
+        if options.any?
+          ActionView.deprecator.warn(<<-MSG.squish)
+            Passing invalid options to content_for is deprecated.
+            Options #{options.keys.join(', ')} will be ignored.
+          MSG
```

Comment:
Is this worth mentioning in the Action View Deprecations section of the [8.2 Release Notes](https://github.com/rails/rails/blob/61b5c7e2953fd0ccfab151b233ce69283c567a34/guides/source/8_2_release_notes.md#L57-L64)?

Reply:
If this change is merged, the next release that it will be part of is 8.2.

In its current form, it will start emitting deprecation warnings. The Release Notes serve as an outward-facing list of behaviors that are deprecated, so adding an entry makes the motivation behind this underlying change more discoverable.

> Also, will this be deprecated in 8.2 and removed in 8.3

If the commit is merged ahead of the 8.2 release, it will be part of the 8.2 release. The timing of its removal (whether 8.3 or later) is up to Core.

Reply:
All right, I'll add it to the `8_2_release_notes`.